### ISSUE TITLE
Missing “Upload” button

### ISSUE THREAD

**Issue by NedusRS** [evansloan/collectionlog.net#30]
TL;DR: there was no button to “Upload Collection Log Data” whenever I downloaded the “Collection Log” plugin - only the two “Reset” buttons. Logging out does not work either, and my username shows up as “undefined” when searched.

I recently installed the “Collection Log” plugin for RuneLite, but I have had some trouble getting my Collection Log to be recorded and submitted. I followed the instructions for the initial recording (open Collection Log in-game, go through every page, log out), and then I tried following the secondary instructions since my account was showing up as “undefined” whenever I used the “Search” function on the Collectionlog.net website. I saw that the instructions mentioned an “Upload Collection Log Data” button on the right-side panel of the plugin, but I am only seeing the two different “Reset” buttons, without any sort of “Upload” button. Logging out also had no effect, even after resetting the plugin, and trying a different computer.

**Comment by NedusRS** [comment]
I have found the issue - I wasn’t checking the plugin library. I’m SO sorry for the dumb spam.
